protos/monocle/http.proto

```diff
@@ -14,6 +14,17 @@ import "monocle/config.proto";
 import "monocle/search.proto";
 import "monocle/crawler.proto";
 import "monocle/user_group.proto";
+import "monocle/login.proto";
+
+service Login {
+  // Check a login username
+  rpc LoginValidation(monocle_login.LoginValidationRequest) returns (monocle_login.LoginValidationResponse) {
+    option (google.api.http) = {
+      post: "/api/2/login/username/validate"
```

Comment:
Why not using a shorter `/api/2/login` instead?

Reply:
Well ok, I'll publish a followup.